### ISSUE TITLE
Quality duplicate in schema

### ISSUE THREAD

**Issue by jaksmid** [openml/OpenML#183]
Rest api returns two different quality elements.
One is from openml.data:
<oml:quality name="NumberOfClasses">6/oml:quality
Second openml.data.qualities.list:
oml:qualityHoeffdingDDM.warnings/oml:quality
These two are incompatible.
Also, response from restapi contains xmlns reference that is not valid: http://openml.org/openml.

**Comment by joaquinvanschoren** [comment]
Sorry, I don't quite understand this issue. Can you rephrase it? Thanks!

**Comment by jaksmid** [comment]
Call to data/qualities/list
returns qualities in the form:
<oml:quality>DecisionStumpKappa</oml:quality>

Call to data/list/
return qualities in the form of:
 <oml:quality name="NumberOfSymbolicFeatures">32</oml:quality>

**Comment by jaksmid** [comment]
Similar thing for Flow list and get flow by id.
Both are returning entities oml:flow (former list of flows, latter single) but every time with different fields.
It is somewhat confusing and inconsistent

**Comment by joaquinvanschoren** [comment]
Still not sure I see the problem. In data/list you want quick access to the values. In data/qualities/list you want more detail on the different qualities. Also see #156.
Please reopen if you think this is still an issue.
